lib/Config/RawConfig.cpp

```diff
@@ -14,6 +14,12 @@ int MullDefaultTimeoutMilliseconds = 3000;
 
 using namespace mull;
 
+static std::string getFileDir(const std::string &filePath) {
+  std::size_t found = filePath.find_last_of("/\\");
+  std::string fileDir = filePath.substr(0, found);
+  return fileDir;
+}
```

Comment:
Would it make sense to use `dirname` (`man 3 dirname`)? There could also be such functions in the `llvm::fs` or `llvm::path` namespace.